vtr_flow/scripts/python_libs/vtr/inspect.py

```diff
@@ -0,0 +1,306 @@
+import re
```

Comment:
This is used for the parse functionality of which I have not tested or looked at. it will be fixed and looked at later.

Reply:
I think we should remove untested code for now.  Can you move parse_vtr_flow into it's own py file and leave it out for now?